pkg/scheduler/cache/event.go

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
it better change the file name from ‘event.go’ to 'event_handlers_interface'

Reply:
golang does not recommend using underscores to name the file name, except for test files
[https://go.dev/doc/effective_go#mixed-caps](url)

Reply:
I know it's a not a recommended way with cunderscores in filenames. From my point of view, an explicit filename with multiple words get more benifit. It's common in k8s. we can shorten it to two words 'event_interface'.

Reply:
modify the filename:
event.go -> eventhandlerinterface.go